acceptance/README.md

```diff
@@ -1,73 +1,97 @@
-source][beaker-hostgenerator].
+Running beaker will then be a matter of using `bundle exec beaker ...` as described
+below.
 
-[beaker-hostgenerator]: https://github.com/puppetlabs/beaker-hostgenerator
+You might also need to obtain the `~/.ssh/id\_rsa-acceptance` file from a
+colleague so that beaker can SSH into the target VMs located in the pooler.
 
-#### Define the PACKAGE_BUILD_VERSION and PUPPET_VERSION environment variables
+### Define your test environment using environment variables
 
-You'll need to provide a couple environment variables that specify which build
-of puppet-server to install and test against.
+You'll need to provide two environment variables that specify which builds of the
+agent and server to install and test against, ```PACKAGE_BUILD_VERSION``` and
+`PUPPET_VERSION`. ```PACKAGE_BUILD_VERSION``` indicates
+the build of the puppet-server package to install.
 
-1. Go to http://builds.delivery.puppetlabs.net/puppetserver/
-2. Scroll down to the most recent build at the bottom.  This will look like:
-   `0.1.4.SNAPSHOT.2014.05.15T1118`
-3. Copy the text (not the link address) - this will be `PACKAGE_BUILD_VERSION`
-4. Define ```PUPPET_VERSION``` as the packaged version of the puppet-agent
-   package that we're depending upon, which is currently `1.1.0`
+1. Go to [http://builds.puppetlabs.lan/puppetserver/?C=M;O=D](http://builds.puppetlabs.lan/puppetserver/?C=M;O=D).
+1. This puts the most recent build at the top of the display. It will look like `.2.2.master.SNAPSHOT.2016.02.04T0323`.
+1. This value (the text, not the link address) is ```PACKAGE_BUILD_VERSION```.
+1. `PUPPET_VERSION` specifies which version of the puppet-agent package to install.
+This is currently `1.3.4`.
 
 If you are testing backwards compaitiblity against a specific release of
-puppet, the `PUPPET_LEGACY_VERSION` environment variable is also available for
+puppet, the ```PUPPET_LEGACY_VERSION``` environment variable is also available for
 this purpose, but is not required.
 
-#### Install dependencies
+### Run Beaker from a script
 
-Beaker needs to be installed and is expressed as a Gemfile dependency.  To
-install beaker, simply run `bundle install --path .bundle/gems` from the top
-level directory of the project.
+In the simplest case, after you have set ```PACKAGE_BUILD_VERSION``` and
+`PUPPET_VERSION`, you can use the script `acceptance/scripts/generic/testrun.sh`,
+which is easier to use than invoking beaker directly. As always, YMMV.
+
+      export PACKAGE_BUILD_VERSION=2.2.2.master.SNAPSHOT.2016.02.02T1619
+      export PUPPET_VERSION=1.3.5
+      acceptance/scripts/generic/testrun.sh -p
 
-Running beaker will then be a matter of `bundle exec beaker ...`
+The `-p` argument tells the script to tell Beaker to preserve the VM hosts
+after testing completes rather than returning them to the pooler. This leaves
+the provisioned and installed VMs ready for further testing. It uses the Beaker
+hosts file.
 
```

Comment:
Insert a mention of using acceptance/scripts/generic/testrun.sh -r to reuse the contents of the host file, to rerun the tests (perhaps after modifying a test)